UnityProject/Assets/Scripts/Food/FoodBehaviour.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking;
+using PlayGroup;
+
+/// <summary>
+/// Food behaviour. The base for every food item in the game
+/// </summary>
+public class FoodBehaviour : NetworkBehaviour {
+
+	//TODO remove after deathmatches
```

Comment:
I will actually make a TODO to remove steak-heal and replace it with gauze and bandages in 0.2.3